packages/lodestar/src/chain/regen/regen.ts

```diff
@@ -141,6 +141,12 @@ export class StateRegenerator implements IStateRegenerator {
         code: RegenErrorCode.ERR_NO_SEED_STATE,
       });
     }
+    if (blocksToReplay.length > 5 * this.config.params.SLOTS_PER_EPOCH) {
```

Comment:
extract 5 to some constant like MAX_EPOCH_TO_PROCESS

Reply:
Consider enforcing this comment programatically with this rule 
**Disallow Magic Numbers (no-magic-numbers)**
https://eslint.org/docs/rules/no-magic-numbers#:~:text=The%20no%2Dmagic%2Dnumbers%20rule,to%20make%20their%20meaning%20explicit.